gcn_classic_text_to_json/notices/calet/__init__.py

```diff
@@ -0,0 +1,112 @@
+import email
+import json
+import os
+
+import requests
+
+from ... import conversion, utils
```

Comment:
is there a missing file that corresponds with this utils import? I see that you are importing convestion and utils from the grandparent package but I am unable to find utils.

Reply:
The file is in #2